src/collection.js

```diff
@@ -60,7 +60,9 @@ export default class Collection {
     this._bucket = bucket;
     this._name = name;
     this._lastModified = null;
-    this._transformers = {remote: []};
+    this._plugins = {
+      remoteTransformers: []
```

Comment:
I left all the IdSchema stuff in #143, so this PR only considers RemoteTransformers.